cli/helpers.go

```diff
@@ -9,6 +9,7 @@ import (
 	v1 "github.com/hashicorp/nomad-openapi/v1"
 	"github.com/hashicorp/nomad-pack/internal/pkg/cache"
 	"github.com/hashicorp/nomad-pack/internal/pkg/errors"
+	intHelper "github.com/hashicorp/nomad-pack/internal/pkg/helper"
```

Comment:
haha, I had to double-take this because I read it as "integer helper" internally and then was surprised by its use.